daemon/graphdriver/fuse-overlayfs/fuseoverlayfs.go

```diff
@@ -175,22 +172,12 @@ func (d *Driver) Create(id, parent string, opts *graphdriver.CreateOpts) (retErr
 
 func (d *Driver) create(id, parent string, opts *graphdriver.CreateOpts) (retErr error) {
 	dir := d.dir(id)
+	root := d.idMap.RootPair()
 
-	rootUID, rootGID, err := idtools.GetRootUIDGID(d.uidMaps, d.gidMaps)
-	if err != nil {
-		return err
-	}
-	root := idtools.Identity{UID: rootUID, GID: rootGID}
-
-	dirID := idtools.Identity{
-		UID: rootUID,
-		GID: rootGID,
-	}
```

Comment:
I noticed that `root == dirID` in this driver so I coalesced the two when I refactored it. But I noticed that other drivers follow a very similar pattern, except that their `root != dirID`: they set `dirID.UID` to `idtools.CurrentIdentity().UID`. Could this be a bug in the fuseoverlayfs driver?

Reply:
^^ @AkihiroSuda

Reply:
Yes, seems a bug

Reply:
Thanks for looking! Ok, given that this PR should not make the situation "worse"; let's fix that separately.

@corhere do you want to work on that? (Otherwise feel free to create a tracking issue for it)